aptos-move/block-executor/src/executor.rs

```diff
@@ -833,7 +836,7 @@ where
         executor_initial_arguments: E::Argument,
         signature_verified_block: &[T],
         base_view: &S,
-    ) -> Result<BlockOutput<E::Output>, PanicOr<IntentionalFallbackToSequential>> {
+    ) -> Result<BlockOutput<E::Output>, ()> {
```

Comment:
We have variety of codification that end parallel execution, and we coalesce them all to have it deterministic (because the real causing condition is hard to get).

So it really is a signal "parallel failure" without more context.